fetch.bs

```diff
@@ -2237,6 +2238,9 @@ the request.
   for HTTP/2, and equivalent information used to prioritize dispatch and processing of
   HTTP/1 fetches.
 
+ <li>
+  <p>If <var>request</var>'s <a for=request>destination</a> is
+  <code>fetch</code>, set it to be the empty string.
```

Comment:
Also, maybe move this step to be before priority, since priority is a later-defined request field.

Looks good with those nits I think. Let me know when there's a WPT PR.